website/common/script/libs/setDebuffPotionItem.js

```diff
@@ -0,0 +1,30 @@
+import getDebuffPotionItem from './getDebuffPotionItem';
+
+
+module.exports = function setDebuffPotionItem (user) {
+  const debuffPotionItems = getDebuffPotionItem(user);
+
+  if (debuffPotionItems.length) {
+    const isUserHaveDebuffInPinnedItems = user.pinnedItems.find(pinnedItem => {
+      let isPresent = false;
+      debuffPotionItems.forEach(debuffPotion => {
+        if (!isPresent) {
+          isPresent = debuffPotion.path === pinnedItem;
+        }
+      });
+      return isPresent;
+    });
+
+    if (!isUserHaveDebuffInPinnedItems) {
+      user.pinnedItems.push(...debuffPotionItems);
```

Comment:
is this going to add duplicates for the debuff potions already in the user pinned items? For example if you should have 4, but have removed 1 from the pinned items this code will add the other 3 again?

Reply:
Good note! I think I need to test it by my self and cover it with unit tests. Thank you :)

Reply:
yeah some unit tests would be great!

Reply:
Added